frontend/src/ui-components/AcmTable/AcmManageColumn.tsx

```diff
@@ -23,6 +23,14 @@ import { IAcmTableColumn } from './AcmTable'
 import { useTranslation } from '../../lib/acm-i18next'
 import ColumnsIcon from '@patternfly/react-icons/dist/js/icons/columns-icon'
 
+type PatternFlySpacingSetting = {
```

Comment:
Don't duplicate type definitions when possible. We can just refer to `ToolbarItemProps['spacer']`. However, I have other suggestions that would avoid needing to add a prop for this at all.